### DISCUSSION TITLE
Tags

### DISCUSSION THREAD

**Discussion by P01NTE45T** [mikekazakov/nimble-commander#315] [Q&A] [answered]
Is there an easy way to (batch) add tags without manually adding them in settings and right-click > tags > add individual tag?

**Comment by mikekazakov** [comment, marked as answer]
There's no specific batch support for than in NC (like e.g. Batch rename), but IIRC MacOS itself has CLI tools for it.
If you want to set tags for many items in once it's possible in NC - simply select multiple files and right-click->tags->pick a tag.
If the files are scattered in a filesystem it's possible to first find all of them via Find Files, then Panelize (Ctrl+P), select and place the tags as usual.
You still need to create the new tag via settings so that NC knows about it, alternatively ask NC to scan the filesystem if it already exists there.
